Update cy_resources.md

aoulas opened the pull request.

bedroesb commented:
Aditional content provided by @aoulas : https://github.com/elixir-europe/rdmkit/pull/1224#issuecomment-1495677703

[https://www.cing.ac.cy/assets/image/imageoriginal/CING_ENG.png]<https://www.cing.ac.cy/>
[https://www.cing.ac.cy/assets/image/imageoriginal/values_blue.png]

[https://www.cing.ac.cy/assets/image/imageoriginal/telethon.png]<https://www.cing.ac.cy/telethon>[https://www.cing.ac.cy/assets/image/imageoriginal/hrexcellence.png]<https://www.cing.ac.cy/en/about-us/fa/good-practices>[https://www.cing.ac.cy/assets/image/imageoriginal/ergodotisisotitas.png]<https://www.cing.ac.cy/en/about-us/fa/good-practices>[https://www.cing.ac.cy/assets/image/imageoriginal/sir.png]<https://www.cing.ac.cy/en/about-us/fa/good-practices>[https://www.cing.ac.cy/assets/image/imageoriginal/bussinessforclimate.png]<https://www.cing.ac.cy/en/about-us/fa/good-practices>[https://www.cing.ac.cy/assets/image/imageoriginal/facebook.png]<https://www.facebook.com/CING.CSMM/>[https://www.cing.ac.cy/assets/image/imageoriginal/twiter.png]<https://twitter.com/NeuroGeneticsCy>[https://www.cing.ac.cy/assets/image/imageoriginal/youtube2.png]<https://www.youtube.com/user/CINGCSMM>

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
 
 ## Funders
+H2020-WIDESPREAD-01-2018-2019: Teaming Phase 2, for €15mi for 7 years. €15mi by the Government of Cyprus and another €8mi by the  University of Cyprus. 
+The Research and Innovation Foundation (RIF) is the national authority in charge of supporting and promoting research, technological development and innovation in Cyprus.
```
It can be difficult for a reader to understand your purpose of mentioning "H2020-WIDESPREAD-01-2018-2019" (additional information and links could be useful). Links are missing here in 'Funders' section e.g. also for RIF.

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
 
 ## Regulations
-<!--- Ethical and legal regulations in the country, committees, etc. --->
+Cyprus National Bioethics Committee (document EEBK3). 
```
Cyprus National Bioethics Committee (document EEBK3) needs a link. Also, is the text after it linked to Bioethics committee or separate text?

aoulas replied:
Added link, thanks!

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
+Cyprus National Bioethics Committee (document EEBK3). 
+All medical records and biological material are kept under secure conditions at a specially prepared 
+Biobank suite with absolute confidentiality and protection of sensitive data of personal character, 
+in compliance with the European General Data Protection Regulation (GDPR)
```
Here you can link GDPR webpage https://gdpr.eu/what-is-gdpr/

aoulas replied:
Done!

Nazeeefa replied:
We will soon have GDPR Compliance page on RDMkit then you can replace the link here with RDMkit webpage link :)

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
-<!--- Ethical and legal regulations in the country, committees, etc. --->
+Cyprus National Bioethics Committee (document EEBK3). 
+All medical records and biological material are kept under secure conditions at a specially prepared 
+Biobank suite with absolute confidentiality and protection of sensitive data of personal character, 
```
You can link here RDMkit page on sensitive data by simply replacing sensitive data with [sensitive data](#sensitive_data)

aoulas replied:
Done!

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
-<!--- Ethical and legal regulations in the country, committees, etc. --->
+Cyprus National Bioethics Committee (document EEBK3). 
+All medical records and biological material are kept under secure conditions at a specially prepared 
+Biobank suite with absolute confidentiality and protection of sensitive data of personal character, 
```
Also, a link to Biobank website for Cyprus needs to be linked here.

aoulas replied:
Added link, thanks!

Nazeeefa commented on pages/national_resources/cy_resources.md:
```diff
@@ -53,12 +52,22 @@ national_resources:
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
+This page provides a general overview of national resources and initiatives on research data management (RDM) in Cyprus
```
If possible, you can also add public webpages of DM-related services and personnel from libraries at the universities in Cyprus.

Nazeeefa commented:
Thanks @aoulas I noticed some links lead to PDF documents, can you please add "(PDF document)" after those links? For example, Cypurs National Policy of the Republic of Cyprus for Open Science Practices (PDF document).

aoulas commented:
> 
Hi Nazeeefa, 

Not sure which steps are still pending for us to get the page online. Can you please help me out. 

Best,

Anastasis

bedroesb commented:
@aoulas I added you as coordinator of this page (the main contact person) Is that ok? I think the page looks OK now. Could you just add some information of the other contributors to this [file](https://github.com/aoulas/rdm-toolkit/blob/patch-6/_data/CONTRIBUTORS.yaml) (optional, but advised)? 

After this I think this page can go live!

aoulas commented:
> > @aoulas I added you as coordinator of this page (the main contact person) Is that ok? I think the page looks OK now. Could you just add some information of the other contributors to this [file](https://github.com/aoulas/rdm-toolkit/blob/patch-6/_data/CONTRIBUTORS.yaml) (optional, but advised)?
> > After this I think this page can go live!
> 
> Hi Bert,
> 
> Many thanks for all the help, I have added additional contributors as requested.
> 
> Best,
> 
> Anastasis

I also just noticed some typos in 2 of the contributors names on the national resources page (cy_resources.md): Nicholas Killilis, Nestoras Kathanasis. The correct names should be: Nicolas Kylilis, Nestoras Karathanasis. Could you please correct these or would you like me to make a new new commit on the page?

bedroesb commented:
Done!

bedroesb commented:
@aoulas the page is live and can be found here: https://rdmkit.elixir-europe.org/cy_resources . Feel free to keep this page up to date/ keep on editing!

aoulas commented:
> @aoulas the page is live and can be found here: https://rdmkit.elixir-europe.org/cy_resources . Feel free to keep this page up to date/ keep on editing!

Thanks again!

A.
